src/components/RelocationBox/RelocationBox.jsx

```diff
@@ -116,8 +177,8 @@ const RelocationBox = ({
       fetchBoxes('pending changes', false),
       fetchBoxes('evaluated', false),
       sendEmail(
-        boxHolderNameState,
-        boxHolderEmailState,
+        getValues().name,
+        getValues().email,
```

Comment:
Can we use formData.name and .email here?

Reply:
Done!